drivers/pinmux/Makefile

```diff
@@ -16,6 +16,7 @@ obj-$(CONFIG_BOARD_STM32F4_DISCO) += stm32/pinmux_board_stm32f4_disco.o
 obj-$(CONFIG_BOARD_STM32F469I_DISCO) += stm32/pinmux_board_stm32f469i_disco.o
 obj-$(CONFIG_BOARD_NUCLEO_F411RE) += stm32/pinmux_board_nucleo_f411re.o
 obj-$(CONFIG_BOARD_NUCLEO_F412ZG) += stm32/pinmux_board_nucleo_f412zg.o
+obj-$(CONFIG_BOARD_STM32F412G_DISCO) += stm32/pinmux_board_stm32f412g_disco.o
```

Comment:
put the pinmux in the board dir

Reply:
Could you be more precise? I can't find other boards with pinmux in the board directories.

Reply:
Look at boards/arm/olimex_stm32_e407/ for example or what I just did in PR #1301